test/unit/control-connection-tests.js

```diff
@@ -58,6 +58,21 @@ describe('ControlConnection', function () {
         done();
       });
     });
+    it('should resolve IPv4 and IPv6 addresses with non default port', function (done) {
+      if (!useLocalhost) {
+        return done();
+      }
+      var cc = new ControlConnection(clientOptions.extend({ contactPoints: ['localhost:9999'] }));
+      cc.getConnection = helper.callbackNoop;
+      cc.refreshOnConnection = helper.callbackNoop;
+      cc.init(function (err) {
+        assert.ifError(err);
+        var hosts = cc.hosts.values();
+        assert.strictEqual(hosts.length, 2);
```

Comment:
Interestingly the tests pass okay locally for me. Other tests fail though.

```
$ npm t

> cassandra-driver@3.2.1 test /home/guy/source/nodejs-driver
> mocha test/unit -R spec -t 5000

  ControlConnection
    #init()
      ✓ should resolve IPv4 and IPv6 addresses with non default port

  1 passing (49ms)
```

Would you expect a developer to be able to clone the repo, run `npm i` and then run `npm t` and for all the tests to pass?